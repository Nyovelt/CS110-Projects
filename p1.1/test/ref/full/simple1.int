addi t0 x0 3
slt a2 t1 t0
sltu a2 t1 t0
