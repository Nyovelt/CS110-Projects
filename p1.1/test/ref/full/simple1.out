.text
00300293
00532633
00533633

.symbol
